src/widgets/index/__tests__/index-test.ts

```diff
@@ -582,6 +582,108 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/index/js/"
       );
     });
 
+    it('inherits from the parent states for the queries', () => {
+      const level0 = index({ indexName: 'level_0_index_name' });
+      const level1 = index({ indexName: 'level_1_index_name' });
+      const level2 = index({ indexName: 'level_1_index_name' });
```

Comment:
The index name wasn't changed on purpose?

Reply:
Nope, will fix it.

Reply:
https://github.com/algolia/instantsearch.js/pull/3937/commits/10d548e16691a1e03ed31b0b565e438aa466f3d2